source/draw/Color.ooc

```diff
@@ -70,6 +71,7 @@ ColorUv: cover {
 		this v = color v
 	}
 	init: func@ (=u, =v)
+	// TODO: use valid default colors
```

Comment:
Going above 255 in RGB is okay but negative amount of light has no equivalence with reality.